swimlane/core/client.py

```diff
@@ -95,11 +103,20 @@ def __init__(
 
         self._session = requests.Session()
         self._session.verify = verify_ssl
-        self._session.auth = SwimlaneAuth(
-            self,
-            username,
-            password
-        )
+
+        if username is not None and password is not None:
+            self._session.auth = SwimlaneJwtAuth(
+                self,
+                username,
+                password
+            )
+        elif access_token is not None:
+            self._session.auth = SwimlaneTokenAuth(
+                self,
+                access_token
+            )
+        else:
+            raise ValueError("Must supply a username/password or access token")
```

Comment:
Might add error when both `user/pass` and `token` are provided that it needs to be one or the other. Minor thing, but should work as is.